### PR TITLE
re-organize testing documents into a subdirectory for organization an…

### PR DESCRIPTION
**Pull request by cpython1234**
…d ease of access

[Link to Jira ticket](https://www.youtube.com/watch?v=dQw4w9WgXcQ)

#Delete comments under headings and replace with appropriate descriptions for each category

## Purpose 
Went to find the tech-x testing documents for reference. realize there were a lot of them and they were seperated from the concept markdown page. Figured adding them all into a folder to help with organization would be helpful.

Might see if it's possible to insert link display into each testing doc for the explanation to the different elements in the testing doc.

## Approach 
i used mkdir and literally made mved the items into it. then i made a commit and pushed.

## Testing
I used my eyes

## Screenshots/Video
(show before/after of the change if possible)

### CODE REVIEW

File: tech_sessions/test_paradigms/test_doubles.md [hunk 1]
```diff
@@ -130,13 +130,6 @@ When using a test double to provide context, you are looking to replicate the co
     when(requests).get(...).thenReturn(response)
     ```
 
-  * In C#, this looks a bit like this:
```
Comment (tpaulshippy) [inline tech_sessions/test_paradigms/test_doubles.md:133]: This is probably still useful. Sorry - should probably keep this.

**Review by tpaulshippy — APPROVED** [review]
Want to leave the Moq example, but other than that, this looks ok.

**Comment by tpaulshippy** [comment]
Here's the link that should be removed (just this line): 
https://github.com/StrongMind/culture/blob/334ad0cd81eb2bdf68dd9286300c535b7fdced17/tech_sessions/test_paradigms/test_fixtures.md?plain=1#L26

File: tech_sessions/test_paradigms/test_doubles.md (hunk 1 above)
Comment (cpython1234) [reply tech_sessions/test_paradigms/test_doubles.md]: no problem at all, I reverted the commit and the code is added back to the test_doubles.md. This is what i see now:

* Mocks set up to respond to a function without specification of what is in the parameters

  * in Python this looks a bit like this: 

    ```python
    # Mockito
    when(requests).get(...).thenReturn(response)
    ```

  * In C#, this looks a bit like this:

    ```c#
    // Moq
    something.Setup(x => x.Add(It.IsAny<string>(), It.IsAny<object>()))
    ```

    The danger of this is you could completely change what the method is asking for, and still pass all your tests

* No assertions are made against anything returning from the mock

  * In Python this looks a bit like this

    ```python
    def test_it_returns_the_status_from_the_api_in_the_result(self):
      when(api_client).get_status().thenReturn('ok')
    	
      result = sut.get_result()
      
      assert type(result) is dict
    ```

**Comment by cpython1234** [comment]
> Here's the link that should be removed (just this line):
> 
> https://github.com/StrongMind/culture/blob/334ad0cd81eb2bdf68dd9286300c535b7fdced17/tech_sessions/test_paradigms/test_fixtures.md?plain=1#L26

Apologies, snuck back in there. Re-removed

**Review by tpaulshippy — APPROVED** [review]

**Pull request merged**